### ISSUE TITLE
Problems compiling, missing code

### ISSUE THREAD

**Issue by XenonFlits** [G6EJD/ESP32-e-Paper-Weather-Display#264]
Hi, 
I like this code very much, but have issues compiling sample file Waveshare_2_13_T5

- Function TitleCase is missing,
- Function obtain_wx_data is missing, 
- Function DecodeWeather is missing,
- Line 56 error " #WeAct 2.13 screen module, you need to change GxEPD2_213_B73 to GxEPD2_213_B74 " 

(note: in recent V2.3.1 of the LilyGO TTGO T5 ESP32 + 2.13inch the DEPG0213BN screen driver is used)

Also it would be helpful to elaborate on exactly what libraries should be loaded (name + author name + version)
I use Arduino IDE 1.8.19

Please update the INO, I would like to continue with this code.

**Comment by G6EJD** [comment]
Just follow the readme to install all the necessary libraries.

**Comment by XenonFlits** [comment]
Thanks for the short answer. I got the message. :-P

It is compiling fine now: 
- I did a complete clean and a reinstall, now using Arduino version 2.3.2 
- Added GxEPD2 1.5.8 by Zingg
- Added the .h files
- Modified the GxEPD2_BW loading (line 55)

The common LiliyGo T5 V2.3.1  2.13 inch E-Paper Display (kind of confusing, these numbers) needs the GxEPD2_213_B74, like the WeAct. The DEPG0213BN display driver chip seems OK with that. 

Thanks again, very happy this works now.
